simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFragmentBookDetail.kt

```diff
@@ -249,9 +249,9 @@ class CatalogFragmentBookDetail : Fragment() {
 
     val status =
       this.bookRegistry.bookOrNull(this.parameters.bookID)
-        ?: this.synthesizeBookWithStatus()
+        ?: this.synthesizeBookWithStatus(this.parameters.feedEntry)
 
-    this.onBookStatusUI(status)
+    this.onBookChangedUI(status)
```

Comment:
This method was renamed solely for consistency with the same method in `CatalogPagedViewHolder`.